opsdir-review-template.md

```diff
@@ -1,16 +1,13 @@
 # ops-dir Review Template
 
-Hello, 
+Hi, 
 
 I have been selected as the Operations Directorate (opsdir) reviewer for this draft. 
 
-The Operations Directorate reviews all operational and management-related drafts to ensure alignment with operational best practices.
+The Operations Directorate reviews Internet-Drafts to ensure alignment with operational best practices and that adequate operational considerations are covered.
 
 More "Guidelines for Considering Operations and Management in IETF Specifications" can be found at https://datatracker.ietf.org/doc/draft-opsarea-rfc5706bis/.
 
-While these comments are primarily for the OPS ADs, the authors should consider them alongside other feedback received. 
```

Comment:
Why remove this statement?

Reply:
Reviews are not only for the OPS ADs. Early reviews are, for example, useful for WG, Chairs, etc. Is there any specific part that we lose in that sentence?